packages/core/src/MiddlewareInterfaceExtension.cpp

```diff
@@ -20,7 +20,24 @@
 #include <experimental/filesystem>
 #include <iostream>
 
+#ifdef _WIN32
+// WINDOWS includes
+#include <Windows.h>
+#include <assert.h>
+#else
 #include <dlfcn.h>
+#endif
+
+#ifdef _WIN32
+#define OPENLIB(libname) LoadLibraryW(libname)
+#define GETLASTERROR() GetLastError()
+#define LIBEXTENSION "dll"
+#else
+#define OPENLIB(libname) dlopen(libname, RTLD_NOW)
+#define GETLASTERROR dlerror()
```

Comment:
Here, the final parenthesis are not needed. If you put it here, you have to remove it in the call later. It works fine when this parenthesis get removed.
```suggestion
#define GETLASTERROR dlerror
```

Reply:
Done